newsletter/tests.py

```diff
@@ -0,0 +1,32 @@
+from audioop import reverse
+from django.test import TestCase
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+from newsletter.forms import NewsletterForm
+from newsletter.models import Newsletter
+from newsletter.views import add_newsletter
+
+class AddNewsletterTest(TestCase):
+    def test_add_newsletter(self):
+        file = SimpleUploadedFile(name='ui_logo.jpg', content=open('newsletter/pictures/ui_logo.jpg', 'rb').read(), content_type='image/jpeg')
+        response = self.client.post('newsletter/add-newsletter/', {
+            'newsletter_text': 'test',
+            'newsletter_picture': file,
+            'newsletter_category': 'tips',
+        })
+        
+
+    def test_get_list_newsletter(self):
+        
+        file = SimpleUploadedFile(name='ui_logo.jpg', content=open('newsletter/pictures/ui_logo.jpg', 'rb').read(), content_type='image/jpeg')
+        response = self.client.post('newsletter/add-newsletter/', {
```

Comment:
Hi Raihan, I think it is good enough but you can reduce the code smell by removing the response. Also don't forget to write the negative cases such as `def test_get_wrong_list_newsletter(self)`
```suggestion
        self.client.post('newsletter/add-newsletter/', {
            'newsletter_text': 'test',
            'newsletter_picture': file,
            'newsletter_category': 'tips',
        })
```